src/VisualStudio/Core/Def/Implementation/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -223,7 +223,7 @@ private void UnregisterGlobalOperationNotifications()
                 // Unilaterally transition us to the finished state.  Once we're finished
                 // we cannot start or stop anymore.
                 _globalNotificationsTask = _globalNotificationsTask.ContinueWith(
-                    _ => GlobalNotificationState.Finished, CancellationToken.None, TaskContinuationOptions.None, TaskScheduler.Default);
+                    _ => GlobalNotificationState.Finished, _shutdownCancellationTokenSource.Token, TaskContinuationOptions.None, TaskScheduler.Default);
```

Comment:
Does this also now mean the Wait() below if it's cancelled will throw an exception?